Packages/ChatSecure/ChatSecure/Classes/CallKitManager/CallManager.swift

```diff
@@ -425,9 +544,12 @@ extension CallManager: CXProviderDelegate {
 	}
 }
 
-extension Notification.Name {
+public extension Notification.Name {
 	public enum CallService {
 		public static let receiveCall = Notification.Name("CallService.receiveCall")
 		public static let endCall = Notification.Name("CallService.endCall")
+		public static let notification = NSNotification.Name.init("notification")
+		public static let changeTypeCall = NSNotification.Name.init("changeTypeCall")
+		public static let changeStatusBusyCall = NSNotification.Name.init("changeStatusBusyCall")
 	}
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/Packages/ChatSecure/ChatSecure/Classes/CallKitManager/CallManager.swift#L555"></span>File should contain 500 lines or less: currently contains 555<br />
<code>file_length</code> <code>CallManager.swift:555</code></td>
    </tr>
  </tbody>
</table>